MonoGame.Framework/Color.cs

```diff
@@ -296,127 +308,98 @@ public Color(float r, float g, float b, float alpha)
             A = (byte)MathHelper.Clamp(alpha * 255, Byte.MinValue, Byte.MaxValue);
         }
 
+        #endregion
+
+        #region Public Properties
+
         /// <summary>
-        /// Gets or sets the blue component.
+        /// The alpha component of this color.
         /// </summary>
         [DataMember]
-        public byte B
+        public byte A

```

Comment:
This is why we avoid re-sorting code in a file.  It makes this diff much more difficult to understand.  You should avoid it in the future.